index.html

```diff
@@ -646,15 +646,18 @@ <h2>Patrocínio</h2>
           </p>
         </div>
 
-        <!-- <div class="row no-gutters supporters-wrap clearfix">
+        <div class="row no-gutters supporters-wrap clearfix">
 
           <div class="col-lg-3 col-md-4 col-xs-6">
             <div class="supporter-logo">
-              <img src="img/supporters/1.png" class="img-fluid" alt="">
+              <a href="https://codeminer42.com" target="_blank">
```

Comment:
I don't think we need a link to sponsor website, but It is up to you.

Reply:
I think it's good to have.